nimbus/src/enrollment.rs

```diff
@@ -846,8 +855,11 @@ mod tests {
         ]
     }
 
-    fn get_experiment_enrollments(db: &Database) -> Result<Vec<ExperimentEnrollment>> {
-        db.collect_all::<ExperimentEnrollment>(StoreId::Enrollments)
+    fn get_experiment_enrollments(
+        db: &Database,
+        writer: &mut Writer,
+    ) -> Result<Vec<ExperimentEnrollment>> {
+        db.get_store(StoreId::Enrollments).collect_all(writer)
```

Comment:
Why does this require a `writer`?

Reply:
You're right, this should take `reader`, good catch!